dashboard/app/models/rubric.rb

```diff
@@ -18,11 +18,16 @@ class Rubric < ApplicationRecord
   belongs_to :lesson
 
   def summarize
+    level_position = lesson.script_levels.filter {|sl| sl.levels.include?(level)}.first&.position
     {
       learningGoals: learning_goals.map(&:summarize),
       lesson: {
         name: lesson.name,
         position: lesson.relative_position,
```

Comment:
Yeah, at least as is. This information is just for the UI, so it would be pretty easy to support lockable levels in the future though.

@tess323 as an fyi